openapi/paths/credit-memos@{id}@transaction-allocations@{transactionId}.yaml

```
@@ -0,0 +1,97 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+  - name: transactionId
+    in: path
+    description: Transaction ID.
```

nitpick: where possible, we should give more detail on ID fields

(I'm not sure if my suggestion is correct here) 

```suggestion
    description: ID of a transaction on which a credit memo has been applied.
```